app/pages/home-for-user/recent-messages.jsx

```diff
@@ -131,24 +135,31 @@ class RecentCollectionsSection extends React.Component {
       this.setState({
         messageAuthors: authorState,
       });
-      return author.get('avatar')
-      .catch(() => {
-        return [];
-      })
-      .then((avatars) => {
-        const avatar = [].concat(avatars)[0]; // Why's this an array?
-        const avatarState = Object.assign({}, this.state.avatars);
-        avatarState[author.id] = avatar;
-        this.setState({
-          avatars: avatarState,
+      if (author) {
+        return author.get('avatar')
+        .catch(() => {
+          return [];
+        })
+        .then((avatars) => {
+          const avatar = [].concat(avatars)[0]; // Why's this an array?
+          const avatarState = Object.assign({}, this.state.avatars);
+          avatarState[author.id] = avatar;
+          this.setState({
+            avatars: avatarState,
+          });
+          return author;
         });
-        return author;
-      });
+      } else {
+        return [];
```

Comment:
I think `author` is an object so this should return `{}` or null. The catch block returns an array because that's the default for a list of avatars.